csrankings-l.csv

```diff
@@ -291,6 +291,7 @@ Leo Bertossi,Carleton University,http://www.scs.carleton.ca/~bertossi,TeBUgNcAAA
 Leo Dorst,University of Amsterdam,https://staff.science.uva.nl/l.dorst,5sjd5F8AAAAJ
 Leo Joskowicz,Hebrew University of Jerusalem,http://www.cs.huji.ac.il/~josko,PoyxglsAAAAJ
 Leo Mark,Georgia Institute of Technology,https://pe.gatech.edu/executive-leadership-team/leo-mark-phd,NOSCHOLARPAGE
+Leo Porter,Univ. of California - San Diego,https://leoporter.ucsd.edu,2ITK0EUAAAAJ

```

Comment:
Name does not correspond to DBLP entry. Closing this PR for failure to follow directions.